kong/tests/test_integration_e2e.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from datadog_checks.kong import Kong
+
+from . import common
+
+
+@pytest.mark.usefixtures('dd_environment')
+def test_check(aggregator, check):
```

Comment:
should this be in test_integration and _assert_check in common?

Reply:
The `_assert_check` is the real test, doesn't really feel right to have it in common.

The integration and e2e test method are just wrapper to run this test in a specific environment. I think it's good to have those 3 related methods (assert method, e2e test, integration test) in the same file, it makes it easier to read/understand (which is one of the most important property IMHO).

Reply:
Grouped everything in one file since we only have few tests.